### PR TITLE
Add iMOD Python training jupyter notebook

### LINKED ISSUE #47: Add jupyter notebook of software days to tutorials sections
For the last Delft Software Days, we've made course material in the form of a jupyter notebook.
So far, we have not added that to the iMOD Documentation.

Quarto supports including Jupyter notebooks, so it shouldn't be too much work to add this.

### PR DESCRIPTION
**Pull request by JoerivanEngelen**
Fixes #47 

Adds jupyter notebook with the Hondsrug example for training. This notebook was developed for the iMOD International Days 2023.

### CODE REVIEW

**Comment by JoerivanEngelen** [comment]
> Thanks Joeri!
> 
> Please make sure to remove the output of the notebook.
> 
> Or simply run `pixi run pre-commit`

I liked the rendered notebooks, because it contains a lot of pictures which makes the tutorial a lot more of attractive to look at. It it alright to preserve it? It makes the file 3MB. I now ignored the specific notebook from typos (and fixed the two correct typos it reported on the way).

UPDATE
--------
Ah, I see now: The notebook is rendered online. This means I have to provide the data with the docs. Probably better, I'll see if I can add it (using git LFS).

**Comment by Huite** [comment]
If you can, skip LFS and just link it somewhere.

**Comment by Hofer-Julian** [comment]
How big is the data and how long does the notebook take to execute?

File: pixi.toml [hunk 1]
```diff
@@ -22,9 +22,13 @@ docs = { cmd = "export QUARTO_PYTHON=python && quarto preview docs", depends_on
...
+imod = ">=0.16.0"
 jupyterlab = "*"
 pre-commit = "*"
 primod = "*"
 quarto = "*"
 quartodoc = "*"
+
+[target.linux-64.dependencies]
+# Install Modflow 6 on linux to render examples on CI.
+modflow6 = ">=6.4.4"
```
Comment (Hofer-Julian) [inline pixi.toml:34]: If you update `pixi.toml`, you also have to update `pixi.lock` :)

Comment (JoerivanEngelen) [reply pixi.toml:34]: Working on it!

Comment (JoerivanEngelen) [reply pixi.toml:34]: It's fixed now!

**Review by Hofer-Julian — APPROVED** [review]
The "Hondsrug" example contains assignments. I am not sure if that's how it's supposed to be. For examples, I typically only include solutions.

If that how it's supposed to be, it's ready to be merged from my side!

**Pull request merged**